docker/crabserver/start.sh

```diff
@@ -39,4 +39,5 @@ if [ "$MODE" = fromGH ]; then
 fi
 
 #==== START THE SERVICE
+export CRYPTOGRAPHY_ALLOW_OPENSSL_102=true
```

Comment:
Should you check openssl version as well here which in fact supports cryptography?

Reply:
So, this export is here because otherwise when cherrypy starts, openssl raises an exception. 
With this export, we convert the exception to a warning (which allows the server to run)

> /data/srv/HG2111d-comp16/sw.dmapelli/slc7_amd64_gcc630/external/py3-cryptography/3.2.1-comp/lib/python3.8/site-packages/cryptography/hazmat/bindings/openssl/binding.py:175: CryptographyDeprecationWarning: OpenSSL version 1.0.2 is no longer supported by the OpenSSL project, please upgrade. The next version of cryptography will completely remove support for it.

I imagine that this is coming from https://github.com/cms-sw/cmsdist/blob/comp_gcc630/py3-cryptography.spec .The [openssl.spec](https://github.com/cms-sw/cmsdist/blob/comp_gcc630/openssl.spec) point to an even earlier version of openssl, so i think that out openssl is coming from the system (cc7, if i am not mistaken).

> Should you check openssl version as well here which in fact supports cryptography?

What woud be the advantage of checking the openssl version? I am pretty sure we are using the system one, why should i check it every time i start the crab REST erver?

The proper solution would be to have a newer version of openssl, but i do not know if this is worth the effort, if anybody else in our group already pt some effort in this direction. Any suggestion is very well appreciated! :)

Thanks for the review, by the way!